File: Android/DaycareParent/DaycareParent/app/src/main/java/com/daycare/daycareparent/model/DailySheetSerializeRequest.java
package com.daycare.daycareparent.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class DailySheetSerializeRequest implements Serializable {
    @SerializedName("id")
    public Integer id;
    @SerializedName("agencyID")
    public Integer agencyID;
    @SerializedName("studentID")
    public Integer studentID;
    @SerializedName("classesID")
    public Integer classesID;
    @SerializedName("activityTypeID")
    public Integer activityTypeID;
    @SerializedName("activityRegisterDate")
    public String activityRegisterDate;

    @SerializedName("selectedStudents")
    public List<Integer> selectedStudents = null;
    @SerializedName("studentActivityMeals")
    public StudentActivityMeals studentActivityMeals;
    @SerializedName("studentActivityMedications")
    public StudentActivityMedications studentActivityMedications;
    @SerializedName("studentActivityNotes")
    public StudentActivityNotes studentActivityNotes;
    @SerializedName("studentActivityMoods")
    public StudentActivityMoods studentActivityMoods;
    @SerializedName("studentOtherActivity")
    public StudentOtherActivity studentOtherActivity;
    @SerializedName("studentAcitivityNap")
    public StudentAcitivityNap studentAcitivityNap;
    @SerializedName("stringId")
    public String stringId;
    @SerializedName("isActive")
    public Boolean isActive;
    @SerializedName("isDeleted")
    public Boolean isDeleted;
    @SerializedName("deletedBy")
    public Integer deletedBy;

    @SerializedName("studentActivitiesID")
    public Integer studentActivitiesID;


    @SerializedName("DeletedDate")
    public String DeletedDate;


    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getAgencyID() {
        return agencyID;
    }

    public void setAgencyID(Integer agencyID) {
        this.agencyID = agencyID;
    }

    public Integer getStudentID() {
        return studentID;
    }

    public void setStudentID(Integer studentID) {
        this.studentID = studentID;
    }

    public Integer getClassesID() {
        return classesID;
    }

    public void setClassesID(Integer classesID) {
        this.classesID = classesID;
    }

    public Integer getActivityTypeID() {
        return activityTypeID;
    }

    public void setActivityTypeID(Integer activityTypeID) {
        this.activityTypeID = activityTypeID;
    }

    public String getActivityRegisterDate() {
        return activityRegisterDate;
    }

    public void setActivityRegisterDate(String activityRegisterDate) {
        this.activityRegisterDate = activityRegisterDate;
    }

    public List<Integer> getSelectedStudents() {
        return selectedStudents;
    }

    public void setSelectedStudents(List<Integer> selectedStudents) {
        this.selectedStudents = selectedStudents;
    }

    public StudentActivityMeals getStudentActivityMeals() {
        return studentActivityMeals;
    }

    public void setStudentActivityMeals(StudentActivityMeals studentActivityMeals) {
        this.studentActivityMeals = studentActivityMeals;
    }

    public StudentActivityMedications getStudentActivityMedications() {
        return studentActivityMedications;
    }

    public void setStudentActivityMedications(StudentActivityMedications studentActivityMedications) {
        this.studentActivityMedications = studentActivityMedications;
    }

    public StudentActivityNotes getStudentActivityNotes() {
        return studentActivityNotes;
    }

    public void setStudentActivityNotes(StudentActivityNotes studentActivityNotes) {
        this.studentActivityNotes = studentActivityNotes;
    }

    public StudentActivityMoods getStudentActivityMoods() {
        return studentActivityMoods;
    }

    public void setStudentActivityMoods(StudentActivityMoods studentActivityMoods) {
        this.studentActivityMoods = studentActivityMoods;
    }

    public StudentOtherActivity getStudentOtherActivity() {
        return studentOtherActivity;
    }

    public void setStudentOtherActivity(StudentOtherActivity studentOtherActivity) {
        this.studentOtherActivity = studentOtherActivity;
    }

    public StudentAcitivityNap getStudentAcitivityNap() {
        return studentAcitivityNap;
    }

    public void setStudentAcitivityNap(StudentAcitivityNap studentAcitivityNap) {
        this.studentAcitivityNap = studentAcitivityNap;
    }

    public String getStringId() {
        return stringId;
    }

    public void setStringId(String stringId) {
        this.stringId = stringId;
    }

    public Boolean getActive() {
        return isActive;
    }

    public void setActive(Boolean active) {
        isActive = active;
    }

    public Boolean getDeleted() {
        return isDeleted;
    }

    public void setDeleted(Boolean deleted) {
        isDeleted = deleted;
    }

    public Integer getDeletedBy() {
        return deletedBy;
    }

    public void setDeletedBy(Integer deletedBy) {
        this.deletedBy = deletedBy;
    }


    public class StudentOtherActivity {

        @SerializedName("subActivityTypeName")
        public String subActivityTypeName;
        @SerializedName("id")
        public Integer id;
        @SerializedName("agencyID")
        public Integer agencyID;
        @SerializedName("studentID")
        public Integer studentID;
        @SerializedName("studentActivitiesID")
        public Integer studentActivitiesID;
        @SerializedName("subActivityTypeID")
        public Integer subActivityTypeID;
        @SerializedName("startTime")
        public String startTime;
        @SerializedName("endTime")
        public String endTime;
        @SerializedName("otherActivityNote")
        public String otherActivityNote;
        @SerializedName("stringId")
        public String stringId;
        @SerializedName("activityTypeID")
        public Integer activityTypeID;
        @SerializedName("isActive")
        public Boolean isActive;

        public String getSubActivityTypeName() {
            return subActivityTypeName;
        }

        public void setSubActivityTypeName(String subActivityTypeName) {
            this.subActivityTypeName = subActivityTypeName;
        }

        public Integer getId() {
            return id;
        }

        public void setId(Integer id) {
            this.id = id;
        }

        public Integer getAgencyID() {
            return agencyID;
        }

        public void setAgencyID(Integer agencyID) {
            this.agencyID = agencyID;
        }

        public Integer getStudentID() {
            return studentID;
        }

        public void setStudentID(Integer studentID) {
            this.studentID = studentID;
        }

        public Integer getStudentActivitiesID() {
            return studentActivitiesID;
        }

        public void setStudentActivitiesID(Integer studentActivitiesID) {
            this.studentActivitiesID = studentActivitiesID;
        }

        public Integer getSubActivityTypeID() {
            return subActivityTypeID;
        }

        public void setSubActivityTypeID(Integer subActivityTypeID) {
            this.subActivityTypeID = subActivityTypeID;
        }

        public String getStartTime() {
            return startTime;
        }

        public void setStartTime(String startTime) {
            this.startTime = startTime;
        }

        public String getEndTime() {
            return endTime;
        }

        public void setEndTime(String endTime) {
            this.endTime = endTime;
        }

        public String getOtherActivityNote() {
            return otherActivityNote;
        }

        public void setOtherActivityNote(String otherActivityNote) {
            this.otherActivityNote = otherActivityNote;
        }

        public String getStringId() {
            return stringId;
        }

        public void setStringId(String stringId) {
            this.stringId = stringId;
        }

        public Integer getActivityTypeID() {
            return activityTypeID;
        }

        public void setActivityTypeID(Integer activityTypeID) {
            this.activityTypeID = activityTypeID;
        }

        public Boolean getActive() {
            return isActive;
        }

        public void setActive(Boolean active) {
            isActive = active;
        }
    }

    public class StudentActivityMedications {

        @SerializedName("id")
        public Integer id;
        @SerializedName("studentID")
        public Integer studentID;
        @SerializedName("agencyID")
        public Integer agencyID;
        @SerializedName("studentActivitiesID")
        public Integer studentActivitiesID;

        @SerializedName("studentMedicationID")
        public Integer studentMedicationID;



        @SerializedName("studentHealthDescription")
        public String studentHealthDescription;
        @SerializedName("recordedTemparture")
        public Integer recordedTemparture;
        @SerializedName("stringId")
        public String stringId;
        @SerializedName("activityTypeID")
        public Integer activityTypeID;
        @SerializedName("isActive")
        public Boolean isActive;
        @SerializedName("isDeleted")
        public Boolean isDeleted;
        @SerializedName("deletedBy")
        public Integer deletedBy;
        @SerializedName("deletedDate")
        public String deletedDate;
        @SerializedName("createdBy")
        public Integer createdBy;

        public Integer getId() {
            return id;
        }

        public void setId(Integer id) {
            this.id = id;
        }

        public Integer getStudentID() {
            return studentID;
        }

        public void setStudentID(Integer studentID) {
            this.studentID = studentID;
        }

        public Integer getAgencyID() {
            return agencyID;
        }

        public void setAgencyID(Integer agencyID) {
            this.agencyID = agencyID;
        }

        public Integer getStudentActivitiesID() {
            return studentActivitiesID;
        }

        public void setStudentActivitiesID(Integer studentActivitiesID) {
            this.studentActivitiesID = studentActivitiesID;
        }

        public String getStudentHealthDescription() {
            return studentHealthDescription;
        }

        public void setStudentHealthDescription(String studentHealthDescription) {
            this.studentHealthDescription = studentHealthDescription;
        }

        public Integer getRecordedTemparture() {
            return recordedTemparture;
        }

        public void setRecordedTemparture(Integer recordedTemparture) {
            this.recordedTemparture = recordedTemparture;
        }

        public String getStringId() {
            return stringId;
        }

        public void setStringId(String stringId) {
            this.stringId = stringId;
        }

        public Integer getActivityTypeID() {
            return activityTypeID;
        }

        public void setActivityTypeID(Integer activityTypeID) {
            this.activityTypeID = activityTypeID;
        }

        public Boolean getActive() {
            return isActive;
        }

        public void setActive(Boolean active) {
            isActive = active;
        }

        public Boolean getDeleted() {
            return isDeleted;
        }

        public void setDeleted(Boolean deleted) {
            isDeleted = deleted;
        }

        public Integer getDeletedBy() {
            return deletedBy;
        }

        public void setDeletedBy(Integer deletedBy) {
            this.deletedBy = deletedBy;
        }

        public String getDeletedDate() {
            return deletedDate;
        }

        public void setDeletedDate(String deletedDate) {
            this.deletedDate = deletedDate;
        }

        public Integer getCreatedBy() {
            return createdBy;
        }

        public void setCreatedBy(Integer createdBy) {
            this.createdBy = createdBy;
        }
    }


    public class StudentActivityNotes {

        @SerializedName("id")
        public Integer id;
        @SerializedName("agencyID")
        public Integer agencyID;
        @SerializedName("studentActivitiesID")
        public Integer studentActivitiesID;
        @SerializedName("studentID")
        public Integer studentID;
        @SerializedName("noteDescription")
        public String noteDescription;
        @SerializedName("stringId")
        public String stringId;
        @SerializedName("activityTypeID")
        public Integer activityTypeID;
        @SerializedName("isActive")
        public Boolean isActive;
        @SerializedName("isDeleted")
        public Boolean isDeleted;
        @SerializedName("deletedBy")
        public Integer deletedBy;
        @SerializedName("deletedDate")
        public String deletedDate;
        @SerializedName("createdBy")
        public Integer createdBy;

        public Integer getId() {
            return id;
        }

        public void setId(Integer id) {
            this.id = id;
        }

        public Integer getAgencyID() {
            return agencyID;
        }

        public void setAgencyID(Integer agencyID) {
            this.agencyID = agencyID;
        }

        public Integer getStudentActivitiesID() {
            return studentActivitiesID;
        }

        public void setStudentActivitiesID(Integer studentActivitiesID) {
            this.studentActivitiesID = studentActivitiesID;
        }

        public Integer getStudentID() {
            return studentID;
        }

        public void setStudentID(Integer studentID) {
            this.studentID = studentID;
        }

        public String getNoteDescription() {
            return noteDescription;
        }

        public void setNoteDescription(String noteDescription) {
            this.noteDescription = noteDescription;
        }

        public String getStringId() {
            return stringId;
        }

        public void setStringId(String stringId) {
            this.stringId = stringId;
        }

        public Integer getActivityTypeID() {
            return activityTypeID;
        }

        public void setActivityTypeID(Integer activityTypeID) {
            this.activityTypeID = activityTypeID;
        }

        public Boolean getActive() {
            return isActive;
        }

        public void setActive(Boolean active) {
            isActive = active;
        }

        public Boolean getDeleted() {
            return isDeleted;
        }

        public void setDeleted(Boolean deleted) {
            isDeleted = deleted;
        }

        public Integer getDeletedBy() {
            return deletedBy;
        }

        public void setDeletedBy(Integer deletedBy) {
            this.deletedBy = deletedBy;
        }

        public String getDeletedDate() {
            return deletedDate;
        }

        public void setDeletedDate(String deletedDate) {
            this.deletedDate = deletedDate;
        }

        public Integer getCreatedBy() {
            return createdBy;
        }

        public void setCreatedBy(Integer createdBy) {
            this.createdBy = createdBy;
        }
    }

    public class StudentActivityMoods {

        @SerializedName("moodTypeName")
        public String moodTypeName;
        @SerializedName("id")
        public Integer id;
        @SerializedName("agencyID")
        public Integer agencyID;
        @SerializedName("studentID")
        public Integer studentID;
        @SerializedName("studentActivitiesID")
        public Integer studentActivitiesID;
        @SerializedName("moodTypeID")
        public Integer moodTypeID;
        @SerializedName("studentMoodDescription")
        public String studentMoodDescription;
        @SerializedName("stringId")
        public String stringId;
        @SerializedName("activityTypeID")
        public Integer activityTypeID;
        @SerializedName("isActive")
        public Boolean isActive;

        public String getMoodTypeName() {
            return moodTypeName;
        }

        public void setMoodTypeName(String moodTypeName) {
            this.moodTypeName = moodTypeName;
        }

        public Integer getId() {
            return id;
        }

        public void setId(Integer id) {
            this.id = id;
        }

        public Integer getAgencyID() {
            return agencyID;
        }

        public void setAgencyID(Integer agencyID) {
            this.agencyID = agencyID;
        }

        public Integer getStudentID() {
            return studentID;
        }

        public void setStudentID(Integer studentID) {
            this.studentID = studentID;
        }

        public Integer getStudentActivitiesID() {
            return studentActivitiesID;
        }

        public void setStudentActivitiesID(Integer studentActivitiesID) {
            this.studentActivitiesID = studentActivitiesID;
        }

        public Integer getMoodTypeID() {
            return moodTypeID;
        }

        public void setMoodTypeID(Integer moodTypeID) {
            this.moodTypeID = moodTypeID;
        }

        public String getStudentMoodDescription() {
            return studentMoodDescription;
        }

        public void setStudentMoodDescription(String studentMoodDescription) {
            this.studentMoodDescription = studentMoodDescription;
        }

        public String getStringId() {
            return stringId;
        }

        public void setStringId(String stringId) {
            this.stringId = stringId;
        }

        public Integer getActivityTypeID() {
            return activityTypeID;
        }

        public void setActivityTypeID(Integer activityTypeID) {
            this.activityTypeID = activityTypeID;
        }

        public Boolean getActive() {
            return isActive;
        }

        public void setActive(Boolean active) {
            isActive = active;
        }
    }
    public class StudentAcitivityNap {

        @SerializedName("id")
        public Integer id;
        @SerializedName("agencyID")
        public Integer agencyID;
        @SerializedName("studentID")
        public Integer studentID;
        @SerializedName("studentActivitiesID")
        public Integer studentActivitiesID;
        @SerializedName("sleptAtTime")
        public String sleptAtTime;
        @SerializedName("workUpTime")
        public String workUpTime;
        @SerializedName("napNote")
        public String napNote;
        @SerializedName("stringId")
        public String stringId;
        @SerializedName("activityTypeID")
        public Integer activityTypeID;
        @SerializedName("isActive")
        public Boolean isActive;

        public Integer getId() {
            return id;
        }

        public void setId(Integer id) {
            this.id = id;
        }

        public Integer getAgencyID() {
            return agencyID;
        }

        public void setAgencyID(Integer agencyID) {
            this.agencyID = agencyID;
        }

        public Integer getStudentID() {
            return studentID;
        }

        public void setStudentID(Integer studentID) {
            this.studentID = studentID;
        }

        public Integer getStudentActivitiesID() {
            return studentActivitiesID;
        }

        public void setStudentActivitiesID(Integer studentActivitiesID) {
            this.studentActivitiesID = studentActivitiesID;
        }

        public String getSleptAtTime() {
            return sleptAtTime;
        }

        public void setSleptAtTime(String sleptAtTime) {
            this.sleptAtTime = sleptAtTime;
        }

        public String getWorkUpTime() {
            return workUpTime;
        }

        public void setWorkUpTime(String workUpTime) {
            this.workUpTime = workUpTime;
        }

        public String getNapNote() {
            return napNote;
        }

        public void setNapNote(String napNote) {
            this.napNote = napNote;
        }

        public String getStringId() {
            return stringId;
        }

        public void setStringId(String stringId) {
            this.stringId = stringId;
        }

        public Integer getActivityTypeID() {
            return activityTypeID;
        }

        public void setActivityTypeID(Integer activityTypeID) {
            this.activityTypeID = activityTypeID;
        }

        public Boolean getActive() {
            return isActive;
        }

        public void setActive(Boolean active) {
            isActive = active;
        }
    }

    public class StudentActivityMeals{
        @SerializedName("id")
        public Integer id;
        @SerializedName("agencyID")
        public Integer agencyID;
        @SerializedName("studentID")
        public Integer studentID;
        @SerializedName("studentActivitiesID")
        public Integer studentActivitiesID;
        @SerializedName("mealTypeName")
        public String mealTypeName;
        @SerializedName("mealTypeID")
        public Integer mealTypeID;
        @SerializedName("mealComment")
        public String mealComment;
        @SerializedName("otherThanPlanMeal")
        public String otherThanPlanMeal;
        @SerializedName("otherThanPlanMealComment")
        public String otherThanPlanMealComment;
        @SerializedName("activityTypeID")
        public Integer activityTypeID;

        @SerializedName("mealPlannerID")
        public Integer mealPlannerID;

        @SerializedName("mealPlanTitle")
        public String mealPlanTitle;

        @SerializedName("studentActivityMealFoodItems")
        public ArrayList<StudentActivityMealFoodItem> studentActivityMealFoodItems=null;



    }
}

File: Android/DayCareTeacher/DayCareTeacher/app/src/main/java/com/daycare/daycareteacher/model/DailySheetSerializeRequest.kt
package com.daycare.daycareteacher.model

import com.google.gson.annotations.SerializedName

import java.io.Serializable
import java.util.ArrayList
import com.google.gson.annotations.Expose



class DailySheetSerializeRequest : Serializable {
    @SerializedName("id")
    var id: Int? = null
    @SerializedName("agencyID")
    var agencyID: Int? = null
    @SerializedName("studentID")
    var studentID: Int? = null
    @SerializedName("classesID")
    var classesID: Int? = null
    @SerializedName("activityTypeID")
    var activityTypeID: Int? = null
    @SerializedName("activityRegisterDate")
    var activityRegisterDate: String? = null

    @SerializedName("selectedStudents")
    var selectedStudents: List<Int>? = null
    @SerializedName("studentActivityMeals")
    var studentActivityMeals: StudentActivityMeals? = null
    @SerializedName("studentActivityMedications")
    var studentActivityMedications: StudentActivityMedications? = null
    @SerializedName("studentActivityNotes")
    var studentActivityNotes: StudentActivityNotes? = null
    @SerializedName("studentActivityMoods")
    var studentActivityMoods: StudentActivityMoods? = null
    @SerializedName("studentOtherActivity")
    var studentOtherActivity: StudentOtherActivity? = null
    @SerializedName("studentAcitivityNap")
    var studentAcitivityNap: StudentAcitivityNap? = null
    @SerializedName("studentActivityDiaper")
    var studentActivityDiaper: StudentAcitivityDiaper? = null


    @SerializedName("stringId")
    var stringId: String? = null
    @SerializedName("isActive")
    var active: Boolean? = null
    @SerializedName("isDeleted")
    var deleted: Boolean? = null
    @SerializedName("deletedBy")
    var deletedBy: Int? = null
    @SerializedName("createdBy")
    var createdBy: Int? = null
    @SerializedName("studentActivitiesID")
    var studentActivitiesID: Int? = null


    @SerializedName("DeletedDate")
    var DeletedDate: String? = null


    inner class StudentOtherActivity {

        @SerializedName("subActivityTypeName")
        var subActivityTypeName: String? = null
        @SerializedName("id")
        var id: Int? = null
        @SerializedName("agencyID")
        var agencyID: Int? = null
        @SerializedName("studentID")
        var studentID: Int? = null
        @SerializedName("studentActivitiesID")
        var studentActivitiesID: Int? = null
        @SerializedName("subActivityTypeID")
        var subActivityTypeID: Int? = null
        @SerializedName("startTime")
        var startTime: String? = null
        @SerializedName("endTime")
        var endTime: String? = null
        @SerializedName("otherActivityNote")
        var otherActivityNote: String? = null
        @SerializedName("stringId")
        var stringId: String? = null
        @SerializedName("activityTypeID")
        var activityTypeID: Int? = null
        @SerializedName("isActive")
        var active: Boolean? = null
    }

    inner class StudentActivityMedications {


        @SerializedName("id")
        @Expose
        var id: Int? = null
        @SerializedName("studentID")
        @Expose
        var studentID: Int? = null
        @SerializedName("agencyID")
        @Expose
        var agencyID: Int? = null
        @SerializedName("studentActivitiesID")
        @Expose
        var studentActivitiesID: Int? = null
        @SerializedName("studentHealthDescription")
        @Expose
        var studentHealthDescription: String? = null
        @SerializedName("recordedTemparture")
        @Expose
        var recordedTemparture: String? = null
        @SerializedName("stringId")
        @Expose
        var stringId: Int? = null
        @SerializedName("activityTypeID")
        @Expose
        var activityTypeID: Int? = null
        @SerializedName("doseRepeatID")
        @Expose
        var doseRepeatID: Int? = null
        @SerializedName("doseRepeatName")
        @Expose
        var doseRepeatName: String? = null
        @SerializedName("dosageQuantityID")
        @Expose
        var dosageQuantityID: Int? = null
        @SerializedName("unit")
        @Expose
        var unit: Int? = null
        @SerializedName("classesID")
        @Expose
        var classesID: Int? = null
        @SerializedName("howTaken")
        @Expose
        var howTaken: String? = null
        @SerializedName("studentMedicationID")
        @Expose
        var studentMedicationID: Int? = null
        @SerializedName("studentMedicationName")
        @Expose
        var studentMedicationName: String? = null
        @SerializedName("isParentAcknowledge")
        @Expose
        var isParentAcknowledge: Boolean? = null
        @SerializedName("isTeacherAcknowledge")
        @Expose
        var isTeacherAcknowledge: Boolean? = null
        @SerializedName("acknowledgeParentID")
        @Expose
        var acknowledgeParentID: Int? = null
        @SerializedName("acknowledgeTeacherID")
        @Expose
        var acknowledgeTeacherID: Int? = null
        @SerializedName("acknowledgeParentName")
        @Expose
        var acknowledgeParentName: String? = null
        @SerializedName("acknowledgeTeacherName")
        @Expose
        var acknowledgeTeacherName: String? = null
        @SerializedName("isMedicationDoneToday")
        @Expose
        var isMedicationDoneToday: Boolean? = null
        @SerializedName("medicationDoneDate")
        @Expose
        var medicationDoneDate: String? = null
        @SerializedName("isActive")
        @Expose
        var isActive: Boolean? = null
        @SerializedName("isDeleted")
        @Expose
        var isDeleted: Boolean? = null
        @SerializedName("deletedBy")
        @Expose
        var deletedBy: Int? = null
        @SerializedName("deletedDate")
        @Expose
        var deletedDate: String? = null
        @SerializedName("createdBy")
        @Expose
        var createdBy: Int? = null
        @SerializedName("createdDate")
        @Expose
        var createdDate: String? = null
        @SerializedName("updatedDate")
        @Expose
        var updatedDate: String? = null
        @SerializedName("updatedBy")
        @Expose
        var updatedBy: Int? = null
        @SerializedName("deletedFromIP")
        @Expose
        var deletedFromIP: String? = null
        @SerializedName("createdFromIP")
        @Expose
        var createdFromIP: String? = null
        @SerializedName("updatedFromIP")
        @Expose
        var updatedFromIP: String? = null

    }


    inner class StudentActivityNotes {

        @SerializedName("id")
        var id: Int? = null
        @SerializedName("agencyID")
        var agencyID: Int? = null
        @SerializedName("studentActivitiesID")
        var studentActivitiesID: Int? = null
        @SerializedName("studentID")
        var studentID: Int? = null
        @SerializedName("noteDescription")
        var noteDescription: String? = null
        @SerializedName("stringId")
        var stringId: String? = null
        @SerializedName("activityTypeID")
        var activityTypeID: Int? = null
        @SerializedName("isActive")
        var active: Boolean? = null
        @SerializedName("isDeleted")
        var deleted: Boolean? = null
        @SerializedName("deletedBy")
        var deletedBy: Int? = null
        @SerializedName("deletedDate")
        var deletedDate: String? = null
        @SerializedName("createdBy")
        var createdBy: Int? = null
    }

    inner class StudentActivityMoods {

        @SerializedName("moodTypeName")
        var moodTypeName: String? = null
        @SerializedName("id")
        var id: Int? = null
        @SerializedName("agencyID")
        var agencyID: Int? = null
        @SerializedName("studentID")
        var studentID: Int? = null
        @SerializedName("studentActivitiesID")
        var studentActivitiesID: Int? = null
        @SerializedName("moodTypeID")
        var moodTypeID: Int? = null
        @SerializedName("studentMoodDescription")
        var studentMoodDescription: String? = null
        @SerializedName("stringId")
        var stringId: String? = null
        @SerializedName("activityTypeID")
        var activityTypeID: Int? = null
        @SerializedName("isActive")
        var active: Boolean? = null
    }


    inner class StudentAcitivityDiaper {
        @SerializedName("id")
        var id: Int? = null
        @SerializedName("agencyID")
        var agencyID: Int? = null
        @SerializedName("studentID")
        var studentID: Int? = null
        @SerializedName("studentActivitiesID")
        var studentActivitiesID: Int? = null
        @SerializedName("diaperChangeTime")
        var diaperChangeTime: String?=null
        @SerializedName("studentActivityDiaperNote")
        var studentActivityDiaperNote: String?=null
        @SerializedName("stringId")
        var stringId: String?=null
        @SerializedName("activityTypeID")
        var activityTypeID: Int? = null
        @SerializedName("isActive")
        var active: Boolean? = null
    }


    inner class StudentAcitivityNap {

        @SerializedName("id")
        var id: Int? = null
        @SerializedName("agencyID")
        var agencyID: Int? = null
        @SerializedName("studentID")
        var studentID: Int? = null
        @SerializedName("studentActivitiesID")
        var studentActivitiesID: Int? = null
        @SerializedName("sleptAtTime")
        var sleptAtTime: String?=null
        @SerializedName("workUpTime")
        var workUpTime: String?=null
        @SerializedName("napNote")
        var napNote: String?=null
        @SerializedName("stringId")
        var stringId: String?=null
        @SerializedName("activityTypeID")
        var activityTypeID: Int? = null
        @SerializedName("isActive")
        var active: Boolean? = null
    }

    inner class StudentActivityMeals {
        @SerializedName("id")
        var id: Int? = null
        @SerializedName("agencyID")
        var agencyID: Int? = null
        @SerializedName("studentID")
        var studentID: Int? = null
        @SerializedName("studentActivitiesID")
        var studentActivitiesID: Int? = null
        @SerializedName("mealTypeName")
        var mealTypeName: String? = null
        @SerializedName("mealTypeID")
        var mealTypeID: Int? = null
        @SerializedName("mealComment")
        var mealComment: String? = null
        @SerializedName("otherThanPlanMeal")
        var otherThanPlanMeal: String? = null
        @SerializedName("otherThanPlanMealComment")
        var otherThanPlanMealComment: String? = null
        @SerializedName("activityTypeID")
        var activityTypeID: Int? = null

        @SerializedName("mealPlannerID")
        var mealPlannerID: Int? = null

        @SerializedName("mealPlanTitle")
        var mealPlanTitle: String? = null

        @SerializedName("studentActivityMealFoodItems")
        var studentActivityMealFoodItems: ArrayList<StudentActivityMealFoodItem>? = null


    }
}
